qingke-rt/Cargo.toml

```diff
@@ -23,8 +23,8 @@ v4 = []
 highcode = []
 
 [dependencies]
-qingke-rt-macros = { path = "./macros", version = "=0.2.1" }
-qingke = { path = "../", version = "=0.2.0" }
+qingke-rt-macros = { path = "./macros", version = "0.3" }
+qingke = { path = "../", version = "0.3" }
```

Comment:
With proper semver this should work fine, if anything breaking has changed, we should bump the second version.